From 9cf13a507a47de584600cd5daa5af6a9846a2d7b Mon Sep 17 00:00:00 2001
From: Ben Walton <bwalton@artsci.utoronto.ca>
Date: Wed, 11 Mar 2009 17:23:35 +0100
Subject: [PATCH] configure: asciidoc version test cleanup

Redirect stderr to /dev/null instead of stdout.  This discards warnings
generated by python 2.6 related to the reorganization of functions within
modules.  The warnings were causing the version detection to break.

Signed-off-by: Ben Walton <bwalton@artsci.utoronto.ca>
---
 configure.ac |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index fe9d7eb..f4b8e49 100644
--- a/configure.ac
+++ b/configure.ac
@@ -291,7 +291,7 @@ fi
 AC_CHECK_PROGS(ASCIIDOC, [asciidoc])
 if test -n "$ASCIIDOC"; then
 	AC_MSG_CHECKING([for asciidoc version])
-	asciidoc_version=`$ASCIIDOC --version 2>&1`
+	asciidoc_version=`$ASCIIDOC --version 2>/dev/null`
 	case "${asciidoc_version}" in
 	asciidoc' '8*)
 		ASCIIDOC8=YesPlease
-- 
1.6.0.5

